licenses/management/commands/publish.py

```diff
@@ -0,0 +1,148 @@
+import os
+import subprocess
+from argparse import ArgumentParser
+
+from django.core.management import BaseCommand
+
+from cc_licenses.settings.base import TRANSLATION_REPOSITORY_DIRECTORY
+
+# Go to cc-licenses-data and do something
+GO_TO_TRANSLATIONS_REPO = f"cd {TRANSLATION_REPOSITORY_DIRECTORY} && "
+PULL_BRANCHES_DOWN_FROM_DEVELOP = "git checkout develop && git pull"
+PULL_DOWN_TRANSLATION_BRANCHES = (
+    GO_TO_TRANSLATIONS_REPO + PULL_BRANCHES_DOWN_FROM_DEVELOP
+)
+
+
+def run_django_distill():
+    """Outputs the build directory
+    """
+    my_env = os.environ.copy()
+    my_env["DJANGO_SETTINGS_MODULE"] = "cc_licenses.settings.publish"
+    cmd = "python manage.py distill-local --quiet --force"
+    return subprocess.run(
+        cmd, shell=True, check=True, text=True, input="YES", env=my_env,
+    )
+
+
+def git_branch_status(branch: str):
+    """Checks if there is a build to commit using git status
+
+    Returns True or False
```

Comment:
Can the docstring be expanded to document in which cases it returns True and in which cases it returns False?

And it's not clear to me just what "if there is a build to commit" means, so maybe that part can be expanded too.

From looking elsewhere at the code, maybe something like "Return True if there are uncommitted changes" would be right? Except then doing a `git checkout` first is kind of confusing, so maybe I'm still misunderstanding the method.

Reply:
Tried to clarify by renaming the definition and updating the docstring.